packages/cli/test/demo/section.js

```diff
@@ -0,0 +1,26 @@
+/**
+ *
+ * @param {*} execa - the command execution environment
+ * @param {(execa: *, testCommand: (command: *, expectation: *) => Promise<true>) => Promise<void>} testRoutine - the test logic implementation
+ * @returns {(t: *) => Promise<void>}
```

Comment:
You should be able to add comments like these at the top of the file to get the execa types, thereby getting rid of the `*` types:
```javascript
/** @import {Execa} from 'execa' */
/** @import {t} from 'ava' */
```

You can find examples of this working successfully elsewhere in the repo.

Reply:
will do